### ISSUE TITLE
Help links

### ISSUE THREAD

**Issue by kprovance** [urvanov-ru/crayon-syntax-highlighter#30]
The help links still point to aramk.com.  The images he used are no longer available, as the service he chose is no longer around.  Perhaps they could be recreated on your site with updated images,..or the wiki for this page?  aramk could pull his blog at any time.

**Comment by urvanov-ru** [comment]
@kprovance Yes, you are right. The links point to aramk.com. I am thinking about moving the articles to my site. It is really good idea.

**Comment by urvanov-ru** [comment]
The real problem is that I don't know how to restore the images. I never saw them. Web Archive didn't save them.  Maybe I could try to create some new images...
